demos/demo-proxy-webapp/pom.xml

```diff
@@ -25,15 +25,6 @@
     </plugins>
   </build>
   <dependencies>
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-slf4j-impl</artifactId>
-      <scope>compile</scope>
```

Comment:
could be test here or add `slf4j-simple` as `test`
Otherwise:
```
mvn test -pl :demo-proxy-webapp
......

[INFO] Running org.eclipse.jetty.demos.ProxyWebAppTest
SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.

```

Reply:
I found a whole bunch more cases with `SLF4J: No SLF4J providers were found.`.
I'll fix them all.